specifications/state_sync/README.md

```diff
@@ -0,0 +1,310 @@
+# State Synchronizer specification
+
+## Abstract
+
+This document describes the spec of a Libra node's state synchronizer component.
```

Comment:
could we give a one liner here ... ", which is responsible for synchronizing the local ledger to the latest state in the LPN or whatever".

then state what this implementation does.